### ISSUE TITLE
Updating Nuget Package

### ISSUE THREAD

**Issue by itlackey** [WordDocX/DocX#109]
I was wondering if you were going to update the nuget package soon. It appears you fixed the Sequence contains no elements exception when setting Table.Design bug in 1.0.0.23, but it is still broken in the 1.0.0.22 version that is available on Nuget.

ref: http://docx.codeplex.com/discussions/650458

Do you have an ETA on when you will update nuget to 1.0.0.23 build?

Thanks for all the hard work!
ITL

**Comment by PrzemyslawKlys** [comment]
I don't think it works... There's unit test I've added that still fails. Until that is fixed there is no fix. 

This still fails.

        [Test]
        public void Test_Table_SetTableDesignNone()
        {
            using (DocX document = DocX.Create(Path.Combine(_directoryDocuments, "TablesDesign.docx")))
            {
                //Add A table                
                Table t = document.AddTable(2, 3);
                Table t1 = document.InsertTable(t);
                t1.Design = TableDesign.None;
                // requires FIX for .None/.Custom set in a row
              //  t1.Design = TableDesign.None;
              //  t1.Design = TableDesign.Custom;
                t1.Design = TableDesign.Custom;
                t1.Design = TableDesign.ColorfulGrid;
                t1.Design = TableDesign.ColorfulGrid;
                t1.MergeCellsInColumn(1, 0, 1);
                t1.InsertRow();
                t1.Rows[2].MergeCells(1, 3);
                t1.InsertRow(3);
                document.Save();
            }
        }

**Comment by itlackey** [comment]
Strange, I forked the code and ran it using the example shown here: http://cathalscorner.blogspot.com/2010/06/docx-and-tables.html
It worked without an error. Have you been able to narrow down where the issue is, because it does not appear to be a problem in simple scenarios. Maybe there is a work around?

**Comment by GrabzIt** [comment]
I love this package, it would be amazing if a new version of the nuget package with the latest changes was released!

**Comment by GlenConway** [comment]
Great library! I agree with @GrabzIt and @itlackey - there are so many other improvements that I think an update to nuget is worthwhile.
